users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+class User(models.Model):
+    user_name  = models.CharField(max_length=40, unique=True, default="")
+    password   = models.CharField(max_length=200)
+    name       = models.CharField(max_length=40)
+    nickname   = models.CharField(max_length=40, null=True)
+    email      = models.CharField(max_length=100, unique=True)
+    contact    = models.CharField(max_length=15)
+    address	   = models.CharField(max_length=100, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
```suggestion
    contact    = models.CharField(max_length=15)
    address	 = models.CharField(max_length=100, null=True)
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)
```
vscode에서는 맞게 보여도 실제로 올라온 코드는 수정이 알 되어 있을 수도 있습니다. 따라서 항상 push 후에는 코드가 잘 올라갔는지 확인해야 합니다. 
vscode에서 수정이 안 되는 것은 vim에서 한 번 수정해 보세요.

Reply:
오 vscode 에서는 정상으로 보였는데 vim에서는 잘못된게 수정이 되네요 ~! 감사합니다 !